medmodels/medrecord/overview.py

```diff
@@ -0,0 +1,135 @@
+    attributes = [col for col in data.columns if col != "id"]
+    attributes.sort()
+    if attributes:
+        for attribute in attributes:
+            if schema and attribute in schema:
+                if schema[attribute][1] == AttributeType.Continuous:
+                    attribute_info = [
+                        f"min: {data[attribute].min()}",
+                        f"max: {data[attribute].max()}",
+                        f"mean: {data[attribute].mean():.{decimal}f}",
+                    ]
+
+                elif schema[attribute][1] == AttributeType.Temporal:
+                    attribute_info = [
+                        f"min: {min(data[attribute])}",
+                        f"max: {max(data[attribute])}",
+                    ]
+
+                elif schema[attribute][1] == AttributeType.Categorical:
+                    categories = data[attribute].unique().sort()
+                    if len(categories) > 5:
+                        attribute_info = [f"{len(categories)} categories"]
+                    else:
+                        category_str = ", ".join(categories)
+                        if len(category_str) < 100:
+                            attribute_info = [
+                                f"Categories: {', '.join(list(categories))}"
+                            ]
+                        else:
+                            attribute_info = [f"{len(categories)} categories"]
+                else:
+                    attribute_info = [f"{len(data[attribute].unique())} unique values"]
+
+            ## Without Schema
+            else:
+                if data[attribute].dtype.is_numeric():
+                    attribute_info = [
+                        f"min: {data[attribute].min()}",
+                        f"max: {data[attribute].max()}",
+                        f"mean: {data[attribute].mean():.{decimal}f}",
+                    ]
+                else:
+                    try:
+                        time_attribute = data[attribute].str.to_datetime()
+                        attribute_info = [
+                            f"min: {min(time_attribute)}",
+                            f"max: {max(time_attribute)}",
+                        ]
+                    except pl.exceptions.ComputeError:
+                        categories = data[attribute].unique().sort()
+                        category_str = ", ".join(categories)
+                        if (len(categories) > 5) | (len(category_str) > 100):
+                            attribute_info = [f"{len(categories)} unique values"]
+                        else:
+                            attribute_info = [f"Values: {', '.join(list(categories))}"]
+
+            data_dict["Attribute"].extend([attribute] * len(attribute_info))
+            data_dict["Info"].extend(attribute_info)
+    else:
+        [data_dict[x].append("-") for x in data_dict.keys()]
+
+    return pl.DataFrame(data_dict)
+
+
+def prettify_table(table_info: pl.DataFrame) -> List[str]:
```

Comment:
I do not quite like this naming, maybe better _embelish_table_

Reply:
_prettify_ is a common term for similar things to this in the programming context :)